apps/api/src/auth/auth.stratergy.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@nestjs/common'
```

Comment:
Change this to `github.stragergy.ts`. Since we will also be having a `google.stratergy.ts`. Also, move this under a `stratergy` folder